ci/format_pre.sh

```diff
@@ -60,9 +60,6 @@ fi
 CURRENT=check_format
 "${ENVOY_SRCDIR}"/tools/code_format/check_format.py fix --fail_on_diff
 
-CURRENT=check_format_go_code
-"${ENVOY_SRCDIR}"/tools/code_format/check_format_go_code.sh
```

Comment:
Just to confirm, we don't need this anymore because we're now using gofmt instead?

Reply:
we are using gofmt in `envoy.code.check`